index.js

```diff
@@ -1,6 +1,21 @@
 /* jshint node: true */
 'use strict';
 
+var VersionChecker = require('ember-cli-version-checker');
+
 module.exports = {
-  name: 'ember-cli-auto-register-helpers'
+  name: 'ember-cli-auto-register-helpers',
+
+  init: function() {
+    var checker = new VersionChecker(this);
+    this._checkerForEmber = checker.for('ember', 'bower');
+  },
+
+  treeFor: function() {
+    if (this._checkerForEmber.lt('2.0.0-beta.1')) {
```

Comment:
Should be `this._checkerForEmber.lt('1.13.0')` as 1.13.0 final includes the dashless helper auto-resolving behavior.